Antidote/LabelView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  LabelView.swift
+//  Antidote
+//
+//  Created by Yoba on 18/12/2016.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+
+/**
+    Kind of normal non-editable label without a border
+ */
+
+class LabelView: NSTextField {
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    init(withFontSize fontSize: CGFloat = CGFloat(12)) {
```

Comment:
There should be no `width` according to Swift guidelines. Also `CGFloat()` can be removed:

```swift
init(fontSize: CGFloat = 12.0) {
```